special-interest.md

```diff
@@ -42,4 +42,13 @@ Community Goal Coordinate work drive change across the Civil Service in the fiel
 
 *Meeting notes:* [Open source meet-up notes](/special-interest/open-source/open-source-meetup-notes)
 
-*Slack Channel:* [#open-code](https://ukgovernmentdigital.slack.com/archives/C0Q3KG7B8) on UK Government Digital slack
\ No newline at end of file
+*Slack Channel:* [#open-code](https://ukgovernmentdigital.slack.com/archives/C0Q3KG7B8) on UK Government Digital slack
+
+---
+### Heads of Engineering
+
+*Aim:* To bring together Heads of Engineering from across government to share best practice, discuss common challenges and work together to improve the quality of software engineering across government. A series of talks and discussions will be held to share knowledge and experience. Supported by the CDDO (Central Digital and Data Office) and a group of volunteers from across government.
+
+*Contact:* cross-government-software-engineering-heads-of-engineering@digital.cabinet-office.gov.uk
```

Comment:
should we put the names of the main organisers here as well? I think it is nice for people to know who is on the "organising group"

Reply:
I debated yes happy to ..